app/assets/src/components/views/SampleUploadFlow/RemoteSampleFileUpload.jsx

```diff
@@ -53,12 +62,26 @@ class RemoteSampleFileUpload extends React.Component {
       }));
 
       this.props.onChange(newSamples);
+
+      logAnalyticsEvent("RemoteSampleFileUpload_connect_succeeded", {
+        projectId: this.props.project.id,
+        bulkPath: this.state.remoteS3Path,
+        newSamples: newSamples.length
+      });
     } catch (e) {
       if (e.status.startsWith("No samples imported")) {
         this.setState({
+          // TODO (gdingle): we should have an error state for
```

Comment:
cc @MarkAZhang

Reply:
I can see the value of having a more specific error message for an invalid bucket path. We'll have to distinguish this case on the back-end. Good idea!